packages/components/calendar/src/calendar-month.tsx

```diff
@@ -54,6 +55,7 @@ export function CalendarMonth(props: CalendarMonthProps) {
               currentMonth={startDate}
               date={date}
               isPickerVisible={isHeaderExpanded}
+              renderCellContent={renderCellContent}
```

Comment:
you can directly retrieve `renderCellContent` from context in `calendar-cell.tsx`. therefore, there is no need to pass it from here.